paasta_tools/kubernetes_tools.py

```diff
@@ -1237,6 +1237,8 @@ def get_kubernetes_metadata(self, git_sha: str) -> V1ObjectMeta:
                 "paasta.yelp.com/service": self.get_service(),
                 "paasta.yelp.com/instance": self.get_instance(),
                 "paasta.yelp.com/git_sha": git_sha,
+                "paasta.yelp.com/deploy_group": self.get_deploy_group(),
+                "paasta.yelp.com/docker_image": self.get_docker_image(),
```

Comment:
with these, we have the exact same set of service dimensions that /status/metrics provides (minus cluster, but I was gonna add that in the prometheus config)